<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editorMainWindow</class>
 <widget class="QMainWindow" name="editorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DoReMi Level Editor v 0.2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxDebug">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Debug Prints</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadSceneButton">
          <property name="text">
           <string>Load Scene Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="unloadSceneButton">
          <property name="text">
           <string>Unload Scene Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetMsgButton">
          <property name="text">
           <string>Reset Message Vectors</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="loadPluginButton">
          <property name="text">
           <string>Load Plugin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="unloadPluginButton">
          <property name="text">
           <string>Unload Plugin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="runFilemapButton">
          <property name="text">
           <string>Run Filemaps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopFilemapsButton">
          <property name="text">
           <string>Stop Filemaps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelFileName">
          <property name="text">
           <string>Export File Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="textBoxFileName"/>
        </item>
        <item>
         <widget class="QPushButton" name="acceptFileNameButton">
          <property name="text">
           <string>Accept</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Selection</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="0">
            <widget class="QPushButton" name="updateTransButton">
             <property name="text">
              <string>Update Transform</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="reloadTransButton">
             <property name="text">
              <string>Reload Transform</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="reloadShapeButton">
             <property name="text">
              <string>Reload Shape</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="updateShapeButton">
             <property name="text">
              <string>Update Shape</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="resetAttButton">
             <property name="text">
              <string>Reset Attributes</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="duplicateAttButton">
             <property name="text">
              <string>Duplicate Attributes</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pasteAttButton">
             <property name="text">
              <string>Paste Attributes</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLineEdit" name="textBoxSelected">
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxSceneInfo">
          <property name="title">
           <string>Scene Info</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelTransforms">
             <property name="text">
              <string>Transforms</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelLights">
             <property name="text">
              <string>Lights</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelMeshes">
             <property name="text">
              <string>Meshes</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="textBoxTransforms">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="textBoxMaterials">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="textBoxMeshes">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelMaterials">
             <property name="text">
              <string>Materials</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="textBoxLights">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
   <zorder></zorder>
   <zorder>groupBoxSceneInfo</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPluginLocation"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionPluginLocation">
   <property name="text">
    <string>Plugin Location...</string>
   </property>
  </action>
  <action name="actionExport_Location">
   <property name="text">
    <string>Export Location...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
